Configuration/TestHost.xcconfig

```diff
@@ -1,13 +1,14 @@
 #include "TestBase.xcconfig"
 
+ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
+CLANG_MODULES_AUTOLINK = NO;
 CODE_SIGN_IDENTITY = "-";
 COPY_PHASE_STRIP = NO;
-LD_RUNPATH_SEARCH_PATHS = $(inherited) @executable_path/Frameworks;
 INFOPLIST_FILE = Realm/Tests/TestHost/Info.plist;
+INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.utilities";
```

Comment:
Building on Xcode cloud was producing a warning that the TestHost app didn't have an app category set. I'm not sure what it's doing differently since I didn't see the warning locally, but setting an arbitrary app category should be harmless since we're not submitting it to the app store anyway.